src/software/ai/evaluation/indirect_chip.cpp

```diff
@@ -135,33 +135,31 @@ std::vector<LegacyTriangle> Evaluation::getAllTrianglesBetweenEnemyPlayers(
     return all_triangles;
 }
 
-std::vector<LegacyTriangle> Evaluation::findOpenTriangles(
-    std::vector<LegacyTriangle> triangles, std::vector<Point> enemy_players)
+std::vector<Triangle> Evaluation::findOpenTriangles(std::vector<Triangle> triangles,
+                                                    std::vector<Point> enemy_players)
 {
-    std::vector<LegacyTriangle> filtered_triangles;
+    std::vector<Triangle> filtered_triangles;
 
     // For every triangle, the 3 points are adjusted so that the robots making up the
     // vertices won't be counted within the triangle, i.e. make every triangle slightly
     // smaller
-    for (LegacyTriangle t : triangles)
+    for (Triangle t : triangles)
     {
+        const std::vector<Point> &tPoints = t.getPoints();
         // Takes vector of triangles from input and adjust every single triangle within it
-        Point p1 =
-            t[0] +
-            ((getTriangleCenter(t)) - t[0]).normalize(2.5 * ROBOT_MAX_RADIUS_METERS);
-        Point p2 =
-            t[1] +
-            ((getTriangleCenter(t)) - t[1]).normalize(2.5 * ROBOT_MAX_RADIUS_METERS);
-        Point p3 =
-            t[2] +
-            ((getTriangleCenter(t)) - t[2]).normalize(2.5 * ROBOT_MAX_RADIUS_METERS);
-
-        LegacyTriangle adjusted_triangle = triangle(p1, p2, p3);
+        Point p1 = tPoints[0] +
+                   (t.center() - tPoints[0]).normalize(2.5 * ROBOT_MAX_RADIUS_METERS);
```

Comment:
Not a blocking comment, very much out-of-scope, but how would people feel about `mean` instead of `center` for a polygon?

Reply:
I'm indifferent

Reply:
How about _centre_? I'm actually indifferent as well

Reply:
Also indifferent

Reply:
Ok, let's change to `mean` then, since it looks like I'm the only one who's not indifferent here. Out-of-scope, I'm happy to make the change later if you don't want to do it here @scveloso, just let me know.

Reply:
Changed it to mean in latest update